ledger/testing/randomAccounts_test.go

```diff
@@ -87,7 +87,6 @@ func TestAccounts(t *testing.T) {
 	zeroValueExceptions := []reflectionhelpers.TypePath{
 		reflectionhelpers.TypePath{}.AddField("MicroAlgos").AddField("Raw"),
 		reflectionhelpers.TypePath{}.AddField("AssetParams").AddMapKey(),
-		reflectionhelpers.TypePath{}.AddField("AssetParams").AddValue().AddField("Total"),
```

Comment:
`zeroValueExceptions` allows the field to always be zero, so we do _less_ testing if line 90 is retained.  Removing it means that our tests do more coverage, they ensure that we try serializing an AssetParams with Total==0 at least once.

I suspect the exception was added with because the author thought an ASA with Total=0 was illegal and because our random account generator never made them like that. But it is legal, and I made sure our `RandomAssetParams` function created them that way.

```
	// Since 0 and 1 Total assets seem extra interesting, make them more often.
	if crypto.RandUint64()%5 != 0 {
		ap.Total = crypto.RandUint64() % 2
	}
```